src/formats/tga.rs

```diff
@@ -0,0 +1,40 @@
+use std::io::{BufRead, Seek, SeekFrom};
+
+use crate::{ImageResult, ImageSize};
+
+pub fn size<R: BufRead + Seek>(reader: &mut R) -> ImageResult<ImageSize> {
+    reader.seek(SeekFrom::Start(0))?;
+
+    let mut header = [0; 18];
+    reader.read_exact(&mut header)?;
```

Comment:
Same as others, just seek to expected offset and use util methods to read dimensions.